### PR TITLE
EWL-5212: Build Policy finder form

### PR DESCRIPTION
**Pull request by designerneil**
## JIRA Ticket(s)
- https://issues.ama-assn.org/browse/EWL-5212

## Description:
This PR adds functionality to the decline button for the Build Policy Finder form 

## To Test:
This PR can only be test in D8. Refer to PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/754
---

## Visual Regressions

n/a

## Relevant Screenshots/GIFs
n/a

## Remaining Tasks
n/a

## Additional Notes
n/a

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Review by damontgomery — DISMISSED** [review]
The `decline` functionality is different on the live site which goes to the homepage if you opened that form in a new window. Currently, if you open this form in a new window, the button does nothing and gives you no feedback.

space

There is styling for the buttons, but they are missing spacing.

**Comment by designerneil** [comment]
@damontgomery you have to load SG2 branch of the same name. The decline will send you back to where you came from

**Comment by designerneil** [comment]
@damontgomery this PR can only be tested on the D8 side

**Comment by damontgomery** [comment]
Copied from chat:

```
For the decline button, this is what happened for me:
- Open a new tab
- Paste the form link in
- Click decline
- .... nothing

- Open a new tab
- Go to any website
- Paste the form link in
- Click decline
- Went back (edited)

On the live site, the first process goes to the homepage.
```

File: styleguide/source/assets/js/webforms.js [hunk 1]
```diff
@@ -58,6 +58,17 @@
...
   }
 
+  // Go back to previous back is user clicks decline submit button
+  $('.ama__button--decline').click(function(e) {
+    e.preventDefault();
+
+    if (document.referrer == "") {
+      document.location.href='/';
+    } else {
+      history.back()
```
Comment (nstriedinger) [inline styleguide/source/assets/js/webforms.js]: @designerneil we're missing some semi-colons and coding standards ('else' should be on a newline

**Review by nstriedinger — APPROVED** [review]

**Pull request merged**